examples/salty-ocean/README.md

```diff
@@ -0,0 +1,53 @@
+# Why is the Ocean Salty? - GenSX Example
```

Comment:
shouldn't we call this "model comparison" or something like that? In a year it seems like people will be confused why there's an example called salty ocean and what it does

Reply:
Updated!